src/IW/Server/Issue.hs

```diff
@@ -0,0 +1,83 @@
+module IW.Server.Issue
+       ( -- * API
+         IssueAPI
+       , issueServer
+
+         -- * Handlers
+       , issueByIdHandler
+       , issuesHandler
+       ) where
+
+import IW.App.Error (WithError)
+import IW.Core.Issue (Issue (..))
+import IW.Db (WithDb, getIssues, getIssueById, getIssuesByLabel)
+import IW.Server.Types (AppServer, ToApi)
+
+data Sort = Id | Title
```

Comment:
Better to have all data types in `Core` modules for consistency.

Reply:
Got it. I was kinda just using this as a temporary prototype to see if this was the right path to go down.